bug-pattern/src/main/java/jp/skypencil/findbugs/slf4j/ManualMessageDetector.java

```diff
@@ -15,7 +15,7 @@
 @CustomUserValue
 public final class ManualMessageDetector extends AbstractDetectorForParameterArray {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This class has 8 parents which is greater than 5 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AMaximumInheritanceDepth)